src/widgets/buttons.ts

```diff
@@ -724,6 +730,59 @@ export function renderAsRow (dom: HTMLDocument, pred: NamedNode, obj: NamedNode,
   return tr
 }
 
+/**
+ * A Div to represent a draggable person, etc in a list
+ *
+ * pred is unused param at the moment
+ */
+export function renderAsDiv (dom: HTMLDocument, pred: NamedNode, obj: NamedNode, options: any): HTMLElement {
+  const div = dom.createElement('div')
+  // add an option to change justify-content...
+  div.setAttribute('style', 'display: flex; align-items: center; justify-content: space-between; height: 2.5em; padding: 1em;')
+
+  options = options || {}
+  const imageDiv = div.appendChild(dom.createElement('div'))
+  const nameDiv = div.appendChild(dom.createElement('div'))
+  const linkDiv = div.appendChild(dom.createElement('div'))
+
+  const image = options.image || faviconOrDefault(dom, obj)
+
+  imageDiv.setAttribute('style', 'width:2.5em; padding:0.5em; height: 2.5em;')
+  linkDiv.setAttribute('style', 'width:2em; padding:0.5em; height: 4em;')
+  imageDiv.appendChild(image)
+
+  if (options.title) {
+    nameDiv.textContent = options.title
+  } else {
+    setName(nameDiv, obj) // This is async
+  }
+
+  if (options.deleteFunction) {
+    deleteButtonWithCheck(dom, linkDiv, options.noun || 'one', options.deleteFunction)
+  }
+  if (obj.uri) {
+    // blank nodes need not apply
+    if (options.link !== false) {
+      const iconLink = linkIcon(dom, obj)
```

Comment:
**suggestion** I think this line should be part of `createLinkForURI`

Reply:
I know what you mean, but I tried to do that and the problem was that LinkIcon is acutally in this button.ts file and I have made a helper file and it would cause recursion unless I moved linkIcon as well... I wasn't sure about moving existing code yet. Do you think it's okay to move the linkIcon function?

Reply:
I remembered this function was also exported and because of that I didn't want to move it in the helper file.  I changed this though to make it cleaner, let me know what you think

Reply:
**suggestion** you could move the function to it's own file and use it from both locations. Then there should not be a circular dependency

Reply:
aww ok, I'll do that.  I have two functions that are like that.

Reply:
I did mange to take the one out (linkIcon), but there are two others that cause me to have to have a couple functions still inside the buttons file. I tried to pull another out (setName), but this uses the store without it being passed in as a parameter. I'm thinking I'm going to leave it at this round. I've already made quite a few changes. In the future I will see about taking this out along with (deleteFunctionWithCheck).